### PR TITLE
Users/lasuredd/bug fix for machines operations

### PR DESCRIPTION
**Pull request by lasuredd**

### CODE REVIEW

File: Tasks/PerformActionOnMachineGroup/Helper.ps1 [hunk 1]
```diff
@@ -41,12 +44,16 @@ function Invoke-OperationHelper
                 $passedOperationCount--
                 Write-Warning(Get-LocalizedString -Key "Operation '{0}' on machine '{1}' failed with error '{2}'" -ArgumentList $operationName, $machine.Name, $operation.Error.Message)
             }
+            else
+            {
+                 Write-Verbose "'$operationName' operation on the machine '$machineName' succeeded" -Verbose
+            }
         }
-
+
```
Comment (abhinitmodi) [inline Tasks/PerformActionOnMachineGroup/Helper.ps1]: remove the tab here

File: Tasks/PerformActionOnMachineGroup/Helper.ps1 [hunk 2]
```diff
@@ -23,13 +23,16 @@ function Invoke-OperationHelper
 
     Foreach($machine in $machines)
     {
+        $machineName = $machine.Name
         $operation = Invoke-OperationOnProvider -machineGroupName $machineGroupName -machineName $machine.Name -operationName $operationName
+        Write-Verbose "[Azure Resource Manager]Call to provider to perform operation '$operationName' on the machine '$machineName' completed" -Verbose
```
Comment (abhinitmodi) [inline Tasks/PerformActionOnMachineGroup/Helper.ps1]: [Azure Resource Manager]Completed the operation $operationName on the machine $machineName?

**Comment by abhinitmodi** [comment]
Signing off

**Comment by pankajkhanzode** [comment]
Signed off

**Comment by NikhilRanjanKumar** [comment]
Signed off

**Pull request merged**